tests/sentry/replays/unit/test_issue_creation.py

```diff
@@ -153,8 +163,34 @@ def test_report_rage_click_no_trace(default_project):
             timestamp=seq1_timestamp.timestamp(),
             url="https://www.sentry.io",
             node={"tagName": "a"},
+            component_name="SmartSearchBar",
             replay_event=mock_replay_event(trace_ids=[]),
         )
 
     # test that the Issue gets created
     assert Group.objects.get(message__contains="div.xyz > a")
+
+
+@pytest.mark.snuba
+@django_db_all
+def test_report_rage_click_no_component_name(default_project):
+    replay_id = "b58a67446c914f44a4e329763420047b"
+    seq1_timestamp = datetime.now() - timedelta(minutes=10, seconds=52)
+    with Feature(
+        {
+            "organizations:replay-click-rage-ingest": True,
+        }
+    ):
+        report_rage_click_issue_with_replay_event(
+            project_id=default_project.id,
+            replay_id=replay_id,
+            selector="div.xyz > a",
+            timestamp=seq1_timestamp.timestamp(),
+            url="https://www.sentry.io",
+            node={"tagName": "a"},
+            component_name=None,
+            replay_event=mock_replay_event(),
+        )
+
+    # test that the Issue gets created with no component name
+    assert Group.objects.get(message__contains="div.xyz > a")
```

Comment:
in this test, instead of asserting the group, can we mock like other above and assert the calls to make sure evidence data is okay like on line 60 here?